mopidy/http/actor.py

```diff
@@ -177,10 +177,23 @@ def _get_static_request_handlers(self):
         return result
 
     def _get_mopidy_request_handlers(self):
+        sites = [app["name"] for app in self.apps + self.statics]
+
+        default_webclient = self.config["http"]["default_webclient"]
+        if default_webclient not in sites:
+            logger.warning(
+                "Invalid default_webclient %s, "
```

Comment:
Again, could/should use f-strings: `f"Invalid default_webclient {default_webclient}, "